src/CSET/operators/regrid.py

```diff
@@ -319,5 +326,41 @@ def regrid_to_single_point(
     regrid_method = getattr(iris.analysis, method, None)
     if not callable(regrid_method):
         raise NotImplementedError(f"Does not currently support {method} regrid method")
-    cube_rgd = cube.interpolate(((lat, lat_pt), (lon, lon_pt)), regrid_method())
+    cube_rgd = cube.interpolate(((lat, lat_tr), (lon, lon_tr)), regrid_method())
     return cube_rgd
+
+
+def transform_lat_long_points(lon, lat, cube):
+    """Transform a selected point in longitude and latitude in ...
+
+    the real world to the corresponding point on the rotated grid
+    of a cube.
+
+    Parameters
+    ----------
+    cube: Cube
+        An iris cube of the data to regrid. As a minimum, it needs to be 2D with
```

Comment:
Is it worth some form of check for dimensions should occur, and an error message raised if the minimum dimensions are not reached?